sdk/python/.python-version

```diff
@@ -0,0 +1 @@
+3.7.6
```

Comment:
Thanks for the change! One more nit, what is this file for?

Reply:
Ah, sorry. That's the file of my environment. Will remove...

Reply:
@ji-yaqi Removed and force pushed the latest commit!